landoui/pages.py

```diff
@@ -1,11 +1,61 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-from flask import Blueprint, render_template
+import os
+
+from flask import (Blueprint, current_app, jsonify, redirect,
+                   render_template, session)
+
+from landoui.app import oidc
 
 pages = Blueprint('page', __name__)
 
 
+def is_logged_in():
+    # TODO:  There surely must be a better/more secure way to establish
```

Comment:
I'm sure there is - or we may have to timestamp their last log in and check it ourselves. For now I think we're ok. Also keep an eye out on https://github.com/mozilla-iam/Flask-pyoidc/pull/1, looks like it might do that for us.